src/routes/v2/projects.rs

```diff
@@ -526,6 +529,21 @@ pub async fn project_edit(
                     )
                     .execute(&mut *transaction)
                     .await?;
+
+                    // On publish event, we send out notification using team *owner* as publishing user, at the time of mod approval
+                    // (even though 'user' is the mod and doing the publishing)
+                    let owner_id =
+                        TeamMember::get_owner_id(project_item.inner.team_id, &mut *transaction)
+                            .await?;
+                    if let Some(owner_id) = owner_id {
+                        insert_project_publish_event(
+                            id.into(),
+                            project_item.inner.organization_id,
+                            owner_id,
+                            &mut transaction,
+                        )
+                        .await?;
+                    }
```

Comment:
hmm, I think this should use the org ID if the project belongs to the org, and if not it should use the owner ID. could it be changed to that?